### ISSUE TITLE
Question: remove eigen and lu methods for stationary distribution?

### ISSUE THREAD

**Issue by spencerlyon2** [QuantEcon/QuantEcon.jl#75]
I was looking over our coverage results and noticed that we don't test `lu_solve` or `eigen_solve` at all.

Is the `gth_solve` algorithm strictly better? If so, I would vote to remove the other two algorithms entirely.

cc: @oyamad @ZacCranko

**Comment by oyamad** [comment]
`gth_solve` should be strictly better at least in terms accuracy.
You might try the following example, for `e = 1e-12` through `1e-17` for example:

```julia
function mc_example(e)
    p = 0.5
    P = [1-(p+e) p       e
         p       1-(p+e) e
         e       e       1-2*e]
    return MarkovChain(P)
end
```

Analytically the unique stationary distribution is `(1/3, 1/3, 1/3)` for any value of `e`.

**Comment by ZacCranko** [comment]
What ever you guys want to do. I thought I included tests for these at one point. Apparently not!

**Comment by spencerlyon2** [comment]
Implemented in #99 see discussion there
